Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -740,13 +749,13 @@ - (void)setBackgroundColor:(UIColor *)newBackgroundColor
   if (shouldApply) {
     CGColorRef oldBackgroundCGColor = _layer.backgroundColor;
     
-    BOOL specialPropertiesHandling = ASDisplayNodeNeedsSpecialPropertiesHandling(checkFlag(Synchronous), _flags.layerBacked);
-    if (specialPropertiesHandling) {
-        _view.backgroundColor = newBackgroundColor;
+    if (_flags.layerBacked) {
+      _layer.backgroundColor = newBackgroundCGColor;
     } else {
-        _layer.backgroundColor = newBackgroundCGColor;
+      _view.backgroundColor = newBackgroundColor;
+      _layer.backgroundColor = _view.backgroundColor.CGColor;
```

Comment:
Same as below if you update the view I don't think you need to update the layer.

Reply:
@nguyenhuy and I were debugging and realized we needed to set both of these. UIView seems to be setting the `_layer.backgroundColor` to `nil` when setting the views background color.

Reply:
We assume these values are bridged but when debugging it seems like the relationship between view and layer is more complex. We're only hitting this now since we are setting backgroundColor directly on the view itself